src/main/java/com/getindata/connectors/http/internal/table/lookup/JavaNetHttpPollingClientFactory.java

```diff
@@ -26,6 +27,7 @@ public JavaNetHttpPollingClient createPollClient(
         return new JavaNetHttpPollingClient(
             httpClient,
             schemaDecoder,
+            new ObjectMapper(),
```

Comment:
In all the example this parameter is `new ObjectMapper()` If there are other cases - then we should test them, otherwise can we remove this parameter and just create the ObjectMapper in the method?

Reply:
Right, I'll create Object Mapper in constructor.